coalib/parsing/FilterHelper.py

```diff
@@ -0,0 +1,38 @@
+from coalib.output.printers.LogPrinter import LogPrinter
+
+from coalib.parsing.filters.LanguageFilter import LanguageFilter
+from coalib.parsing.filters.CanDetectFilter import CanDetectFilter
+from coalib.parsing.filters.CanFixFilter import CanFixFilter
+
+
+class FilterHelper:
+    all_filters = {'language': LanguageFilter,
+                   'can_detect': CanDetectFilter,
+                   'can_fix': CanFixFilter}
+
+    @classmethod
+    def get_all_filters_str(cls, sep=', '):
+        return sep.join(sorted(cls.all_filters))
```

Comment:
I think you should inline this into the `argparse` code above, imo this is usually application specific code, having a function here is not so modular imo :+1: